optimize/self-managed/optimize-deployment/migration-update/3.11_8.3-to-3.12_8.4.md

```diff
@@ -0,0 +1,119 @@
+---
+id: 3.11_8.3-to-3.12_8.4
+title: "Update notes (8.3/3.11 to 8.4/3.12)"
+---
+
+:::note Heads up!
+To update Optimize to version 8.4/3.12, perform the steps in the [migration and update instructions](./instructions.md).
+:::
+
+The update to 8.4/3.12 can be performed from any 8.3/3.11 release.
+
+Here you will find information about:
+
+- Limitations
+- Known issues
+- Changes in supported environments
+- Changes in behavior (for example, due to a new feature)
+- Changes in translation resources
+
+## Changes in supported environments
+
+### Elasticsearch
+
+With 8.4/3.12, Optimize now supports Elasticsearch `8.9`.
```

Comment:
Everything below this line all the way down to the translations sections is not needed, it was only relevant for the update to 3.11 :)